giskard_vision/core/dataloaders/base.py

```diff
@@ -4,8 +4,19 @@
 
 import numpy as np
 
+from giskard_vision.core.detectors.base import IssueGroup
+
 from ..types import TypesBase
 
+EthicalIssueMeta = IssueGroup(
```

Comment:
I also moved them somewhere in https://github.com/Giskard-AI/giskard-vision/pull/49

Let's merge your PR first

Reply:
ok